StreamChatClient_v3/APIClient/Endpoints/ChannelEndpoints.swift

```diff
@@ -45,7 +45,7 @@ struct ChannelPayload<ExtraData: ExtraDataTypes>: Decodable {
     init(from decoder: Decoder) throws {
         let container = try decoder.container(keyedBy: CodingKeys.self)
         channel = try container.decode(ChannelDetailPayload<ExtraData>.self, forKey: .channel)
-        watcherCount = try container.decode(Int.self, forKey: .watcherCount)
+        watcherCount = try container.decodeIfPresent(Int.self, forKey: .watcherCount) ?? 0
```

Comment:
I also have this change on my PR, I think `watcherCount` should be optional

Reply:
OK, removing this from my PR